<launch>
    <!-- X Supplies -->
    <node name="PSU0" pkg="ros_coils" type="PSU_node" output="screen">
        <param name="PSU_COM" type="str" value="/dev/ttyUSB0"/>
        <param name="vConv" type="double" value="0.1"/>
        <param name="debugMode" type="bool" value="true"/> <!-- debugMode="true" -->
    </node>
    <node name="PSU1" pkg="ros_coils" type="PSU_node" output="screen">
        <param name="PSU_COM" type="str" value="/dev/ttyUSB1"/>
        <param name="vConv" type="double" value="0.1"/>
        <param name="debugMode" type="bool" value="true"/> <!-- debugMode="true" -->
    </node>

    <!-- Z Supplies -->
    <node name="PSU2" pkg="ros_coils" type="PSU_node" output="screen">
        <param name="vConv" type="double" value="0.01"/>
        <param name="PSU_COM" type="str" value="/dev/ttyUSB2"/>
        <param name="debugMode" type="bool" value="true"/> <!-- debugMode="true" -->
    </node>
    <node name="PSU3" pkg="ros_coils" type="PSU_node" output="screen">
        <param name="vConv" type="double" value="0.01"/>
        <param name="PSU_COM" type="str" value="/dev/ttyUSB3"/>
        <param name="debugMode" type="bool" value="true"/> <!-- debugMode="true" -->
    </node>    
    
    <!-- Y Supplies -->
    <node name="PSU4" pkg="ros_coils" type="PSU_node" output="screen">
        <param name="vConv" type="double" value="0.01"/>
        <param name="PSU_COM" type="str" value="/dev/ttyUSB4"/>
        <param name="debugMode" type="bool" value="true"/> <!-- debugMode="true" -->
    </node>
    <node name="PSU5" pkg="ros_coils" type="PSU_node" output="screen">
        <param name="vConv" type="double" value="0.01"/>
        <param name="PSU_COM" type="str" value="/dev/ttyUSB5"/>
        <param name="debugMode" type="bool" value="true"/> <!-- debugMode="true" -->
    </node>
    
</launch>